climada/util/coordinates.py

```diff
@@ -1491,8 +1492,9 @@ def read_raster(file_name, band=None, src_crs=None, window=None, geometry=None,
         number of lons for transform
     height : float
         number of lats for transform
-    resampling : rasterio.warp.Resampling optional
-        resampling function used for reprojection to dst_crs
+    resampling : int, rasterio.enums.Resampling or str, optional
```

Comment:
I haven't used `Resampling` much. What happens when this parameter is an integer?

Reply:
The enum `rasterio.enums.Resampling` actually has integer values (https://rasterio.readthedocs.io/en/latest/api/rasterio.enums.html#rasterio.enums.Resampling), so the type of function parameters is `int, rasterio.enums.Resampling`, see e.g. the docstring of `rasterio.warp.reproject` (https://rasterio.readthedocs.io/en/latest/api/rasterio.warp.html#rasterio.warp.reproject). In addition to that, and for convenience, we also support the string literals.

Reply:
oh nice